src/ontology/pato-edit.obo

```diff
@@ -19871,6 +19871,22 @@ is_a: PATO:0000051 ! morphology
 property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0002-7073-9172
 creation_date: 2021-01-23T11:31:53Z
 
+[Term]
+id: PATO:0010007
+name: multiciliated
+def: "A cellular quality inhering in a bearer by virtue of the bearer's having multiple cilia." [PMID:36924208]
+is_a: PATO:0001408 ! ciliated
+property_value: http://purl.org/dc/terms/date "2024-07-23T10:43:20Z" xsd:dateTime
+property_value: http://www.geneontology.org/formats/oboInOwl#http://purl.org/dc/terms/contributor https://orcid.org/0000-0002-7073-9172
```

Comment:
This annotation uses a combination of oio and terms. @Caroline-99, could you please create a PR to fix it? The same happens in the following term: PATO:0010008.
Change the annotation to `http://purl.org/dc/terms/contributor`

Reply:
Done